### ISSUE TITLE
Incorrect local import

### ISSUE THREAD

**Issue by Dexagod** [shapetrees/test-suite#11]
Hi,

There seems an incorrect import in the shapetree.js module in the shapetree.js/lib/shape-tree.js file.
https://github.com/shapetrees/shapetree.js/blob/d20e57f5f822012a7974a1730becca7de61771b5/lib/shape-tree.js#L26

This import looks for a non-existing local package.
Probably this import should be for @shexjs/path?

I changed this locally and now the tests run for me.

**Comment by ericprud** [comment]
Yep, gotta put that on my list of stuff to update when Jose and I are done finalizing the path language (which we're working on now).
